src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -557,23 +601,26 @@ public MaterialStore getMaterialStore()
     }
 
     /**
-     * The AbstractBuilding View is the client-side representation of a AbstractBuilding.
-     * Views contain the AbstractBuilding's data that is relevant to a Client, in a more client-friendly form
-     * Mutable operations on a View result in a message to the server to perform the operation
+     * The AbstractBuilding View is the client-side representation of a
+     * AbstractBuilding. Views contain the AbstractBuilding's data that is
+     * relevant to a Client, in a more client-friendly form Mutable operations
+     * on a View result in a message to the server to perform the operation
      */
     public static class View
     {
-        private final   ColonyView       colony;
-        private final   BlockPos location;
+        private final ColonyView colony;
+        private final BlockPos   location;
 
-        private int     buildingLevel       = 0;
-        private int     buildingMaxLevel    = 0;
+        private int buildingLevel    = 0;
+        private int buildingMaxLevel = 0;
 
         /**
          * Creates a building view.
          *
-         * @param c ColonyView the building is in.
-         * @param l The location of the building.
+         * @param c
+         *            ColonyView the building is in.
+         * @param l
+         *            The location of the building.
          */
         protected View(ColonyView c, BlockPos l)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Protected method '<init>(...)' is never used. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUnusedProtectedMethod)